xds/internal/xdsclient/transport/transport.go

```diff
@@ -428,13 +453,19 @@ func (t *Transport) recv(stream adsStream) bool {
 	for {
 		resources, url, rVersion, nonce, err := t.recvAggregatedDiscoveryServiceResponse(stream)
 		if err != nil {
-			t.adsStreamErrHandler(err)
+			// If we have already successfully received a message on the stream, then
+			// the OnError callback is called with an error of type IgnoredADSRecvError.
```

Comment:
This comment is simply saying what is happening in the next few lines. You can either remove the comment, or say why it is safe to ignore these errors. Again, you could adapt the verbiage from A57:

```
Note that we do not consider it an error if the ADS stream was closed after having received a response on the stream. This is because there are legitimate reasons why the server may need to close the stream during normal operations, such as needing to rebalance load or the underlying connection hitting its max connection age limit (see [gRFC A9](https://github.com/grpc/proposal/blob/master/A9-server-side-conn-mgt.md)).
```

Reply:
thank! I'll keep a note of that